flag.go

```diff
@@ -1056,6 +1058,7 @@ func (f *FlagSet) Parse(arguments []string) error {
 		case ContinueOnError:
 			return err
 		case ExitOnError:
+			fmt.Println(err)
```

Comment:
Actually, it's a bad idea to integrate it. User should handle error by himself/herself (use log or fmt). And with this spf13/cobra prints error twice. Please delete it.

Reply:
How can Cobra print after os.Exit is called? I agree that the user should be able to decide, but I assumed that after os.Exit had been called, there was no way to show the error anymore.

Reply:
> Exit causes the current program to exit with the given status code. Conventionally, code zero indicates success, non-zero an error. The program terminates immediately; deferred functions are not run. 

So I believe without that line, all prints would be gone. I think Cobra would use ContinueOnError, so this PR would not make it print anything.

Reply:
Oh, it was my dumb fault 😓
You're right.
Anyway cobra uses ContinueOnError as errorHandling.